src/AspNet.Security.OAuth.Validation/Events/IValidationEvents.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Threading.Tasks;
+
+namespace AspNet.Security.OAuth.Validation.Events
+{
+    public interface IValidationEvents
```

Comment:
`IValidationEvents` is too generic. Consider renaming it to `IOAuthValidationEvents`